src/pages/BookmarkPage.js

```diff
@@ -43,12 +43,18 @@ export default function BookmarkPage({
 const Header = styled.h2`
   display: flex;
   justify-content: center;
-  font-size: 3rem;
+  font-size: 1.5rem;
   color: #5c9875;
 `;
 const ListWrapper = styled.ul`
   margin: 0rem 0rem 5rem -2.5rem;
   list-style: none;
+  @media (min-width: 650px) {
```

Comment:
bestimmt richtig so! Ich weiß nur nicht ob du das unbedingt brauchst, da wir ja sowieso nur für das Handy Format programmieren...

Reply:
Just a suggestion, have a look at [breakpoints](https://tailwindcss.com/docs/responsive-design)

Reply:
Thank you both for the feedback! :)